imports/server/migrations/41-document-activities.ts

```diff
@@ -8,7 +7,5 @@ Migrations.add({
   async up() {
     await DocumentActivities.createIndexAsync({ hunt: 1 });
     await DocumentActivities.createIndexAsync({ document: 1, ts: 1 }, { unique: true });
-    await DocumentWatches.createIndexAsync({ document: 1 }, { unique: true });
```

Comment:
In general, how do we want to deal with collections that we explicitly abandon?  Would it make sense to add a migration with a `collection.drop()` here?

Reply:
We have historically just abandoned them and not explicitly cleaned them up. (You can see this in production with `jr_document_perms`, which was superseded by `jr_folder_perms` after #568 and with `jr_profiles` which was rolled into `users)

My general rule of thumb is that dropping a collection is irreversible (which makes the commits irrevertable) and scary, and none of our data is big enough that this is a major concern. But I could be convinced otherwise

Reply:
I can see where you're coming from, though I think that in general, our migrations are not safely revertable (and accordingly, we should be careful with what we put in them).  We do sometimes stage our rollouts to make them at least still smoothly operable at both version N-1 and version N.

I don't think we need to eagerly drop collections specifically?  I'm just thinking about the possibility that future us decide to make a collection which happens to be named the same as one that we have long since forgotten about, but where the production DBs still have data (possibly in a different schema) and that creates trouble (e.g. a migration creates a unique index on a key that the prod DB contents lack, or the leftover indexes prevent inserting new records with different contents).  I guess we could also do something like keep tombstone collections around so we'd be forced to think about what to do with the dangling records if we ever try to return to them later?

The upside of dropping the collection is that we don't have to remember the constraints imposed by the previously-abandoned data.  Maybe it'd be reasonable for us to clean these abandoned collections up after some purgatory period (a few months?  one Hunt?)

We can totally solve that in a separate PR, though.